autosklearn/automl.py

```diff
@@ -171,6 +171,27 @@ def __init__(self,
             self._resampling_strategy_arguments['folds'] = 5
         self._n_jobs = n_jobs
         self._dask_client = dask_client
+
+        # If no dask client was provided, we create one, so that we can
+        # start a ensemble process in parallel to smbo optimize
+        if self._dask_client is None:
+            processes = False
+            if self._n_jobs is not None and self._n_jobs > 1:
+                processes = True
+                dask.config.set({'distributed.worker.daemon': False})
+            self._dask_client = dask.distributed.Client(
+                n_workers=self._n_jobs,
+                processes=processes,
+                threads_per_worker=1,
+                local_directory=self._backend.temporary_directory,
+            )
+
+        # For the ensemble building process
+
+        # How much patience to wait for the ensemble building process to finish by itself
+        self.ensemble_timeout_patience = 10
```

Comment:
When is this patience actually used in practice? Does this happen often?

Reply:
It is a safety measure. Often the even killer is the one who ends the ensemble builder. But if something happened in the worker, that lost the job for some very bad reason, this is a safety measure so that people don't complain that autosklearn hangs up